front_end/core/react_native/generated/type_defs/ReactNativeChromeDevToolsDebuggerEventTypes.d.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ * Copyright 2024 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ *
+ * @generated by xplat/js/react-native-github/scripts/debugger-frontend/gen-type-defs.js
```

Comment:
Whats the latest on that? Will we sync these files manually for now?

Reply:
Yeah friendship ended with with code gen, ⌘ + C is now my best friend